### DISCUSSION TITLE
Domyos BIKE 500 & 500SP => Android => Zwift

### DISCUSSION THREAD

**Discussion by YoMhz** [cagnulein/qdomyos-zwift#2456] [Q&A]
Hello, I have just purchased the qdomyos-zwift application on android. I can connect the application to Zwift without any problems. However, I find that the sensations do not correspond. During the training phases it asks me to push in Watts. For example, to reach 350 W, I pedal at a high cadence and it's very difficult to reach 300 W when I'm at MAX power.

Here's my bike model: 

https://support.decathlon.fr/bike-500

Thanks for your help!

**Comment by cagnulein** [comment]
hi, is the resistance at max level?

**Reply by YoMhz** [reply]
The basic cycle goes from 1 to 12. On a training session, it went to 15.
I was sprinting and it was impossible to reach the wattage requested by zwift. In terms of animation, the other cyclists are quietly overtaking me, while I'm always out and about (in the game) and really into it in real life haha!

**Reply by cagnulein** [reply]
you can try to enable the domyos 500 wattage profile in the domyos bike settings in qz. Anyway your bike doesn't offer a level of the resistance so high to reach higher wattage. 350 it's already a good wattage

**Reply by YoMhz** [reply]
I've just activated it! I'll try this tomorrow. Thanks for your very quick reply! I'll be back if I have any more questions! THANK YOU SO MUCH! ;)

**Reply by cagnulein** [reply]
remember to rate the app on the store, it means a lot to me! also if you can join the swag bag from the app to support the app!
Thanks

**Reply by YoMhz** [reply]
No worries. I'll give myself some time to test it out and get back to you!
